### ISSUE TITLE
DB/Locale/ruRU Missing Texts (Part 3)

### ISSUE THREAD

**Issue by Freeman256** [TrinityCore/TrinityCore#26632]
```sql
-- Correcting almost all quests in Dustwallow Marsh for the 32-38 Alliance. / Исправление почти всех квестов в Пылевых Топях для Альянса 32-38 уровня.
-- This time, taking into account the comments of **offl**, for which many thanks to him. / На этот раз с учётом замечаний **offl**, за что ему большое спасибо.
UPDATE `quest_template_locale` SET `Title`='Мрачные выводы' WHERE  `ID`=11143 AND `locale`='ruRU';
UPDATE `quest_template_locale` SET `Title`='Оправдавшиеся подозрения' WHERE  `ID`=11144 AND `locale`='ruRU';
UPDATE `quest_template_locale` SET `Title`='Рука Зловещего Тотема' WHERE  `ID`=11148 AND `locale`='ruRU';
UPDATE `quest_template_locale` SET `Title`='Помощь Табеты' WHERE  `ID`=11149 AND `locale`='ruRU';
UPDATE `quest_template_locale` SET `ObjectiveText1`='Убито тауренов из племени Зловещего Тотема' WHERE  `ID`=11156 AND `locale`='ruRU';
UPDATE `quest_template_locale` SET `Title`='Письмо аптекаря' WHERE  `ID`=11185 AND `locale`='ruRU';
UPDATE `quest_template_locale` SET `ObjectiveText1`='Тотем испытан' WHERE  `ID`=11169 AND `locale`='ruRU';
UPDATE `quest_template_locale` SET `Title`='Подмена реактива' WHERE  `ID`=11173 AND `locale`='ruRU';
UPDATE `quest_template_locale` SET `Title`='Сожги заставу Дикого рога!' WHERE  `ID`=11150 AND `locale`='ruRU';
UPDATE `quest_template_locale` SET `ObjectiveText1`='Северный шатер подожжен', `ObjectiveText2`='Северо-восточный шатер подожжен', `ObjectiveText3`='Восточный шатер подожжен' WHERE  `ID`=11150 AND `locale`='ruRU';
UPDATE `quest_template_locale` SET `Title`='Отмщение за семейство Хьяль' WHERE  `ID`=11151 AND `locale`='ruRU';
UPDATE `quest_template_locale` SET `ObjectiveText1`='Уничтожено слизнюков' WHERE  `ID`=11174 AND `locale`='ruRU';
UPDATE `quest_template_locale` SET `Title`='Знамя Каменного Молота' WHERE  `ID`=11160 AND `locale`='ruRU';
UPDATE `quest_template_locale` SET `Title`='Сущность вражды' WHERE  `ID`=11161 AND `locale`='ruRU';
UPDATE `quest_template_locale` SET `Title`='Тяни змею за хвост!' WHERE  `ID`=11217 AND `locale`='ruRU';
UPDATE `quest_template_locale` SET `Title`='Духи в руинах деревни Каменного Молота', `ObjectiveText1`='Духов Каменного Молота упокоено' WHERE  `ID`=11159 AND `locale`='ruRU';
UPDATE `quest_template_locale` SET `Title`='Бросить вызов черным драконам', `ObjectiveText1`='Клан Каменного Молота отмщен' WHERE  `ID`=11162 AND `locale`='ruRU';
UPDATE `quest_template_locale` SET `Title`='Обрести покой', `ObjectiveText1`='Венок к памятнику семейства Хьяль возложен' WHERE  `ID`=11152 AND `locale`='ruRU';

-- Correction of the dragon text in quest 11162. Many thanks to **offl** for his help. / Исправление текста дракона в квесте 11162. Большое спасибо **offl** за его помощь.
UPDATE `smart_scripts` SET `target_type` = 23 WHERE `entryorguid` = 2378900 AND `source_type` = 9 AND `id` IN (1,2);

DELETE FROM `quest_greeting_locale` WHERE `ID` IN (3391, 3453) AND `locale` = 'ruRU';
INSERT INTO `quest_greeting_locale` (`ID`, `locale`, `Greeting`) VALUES
(3391, 'ruRU', 'Тралл хорошо заплатил мне и моим парням за помощь в строительстве Оргриммара, поэтому я решил построить здесь порт. Мы ведем большую часть нашего бизнеса через Пиратскую Бухту и Барона Ревилгаза.'),
(3453, 'ruRU', 'Чем я могу вам помочь, $n? Если ничем, то я должен позаботиться об отправке.');

DELETE FROM `quest_request_items_locale` WHERE `ID` IN (1111,1112,1114,11144,11148,11173,11169,11150,11185,11174,11158,11160,11161,11217,11159,11162,11152) AND `locale` = 'ruRU';
INSERT INTO `quest_request_items_locale` (`ID`, `locale`, `CompletionText`) VALUES 
(1111, 'ruRU', 'Следующий! Так, ты отправитель или получатель?'),
(1112, 'ruRU', 'Вы уже забрали части из Кабестана?'),
(1114, 'ruRU', 'Принес мне что-нибудь? Здорово! Ну-ка, посмотрим...'),
(11144, 'ruRU', 'Удалось найти какие-нибудь следы приказов?'),
(11148, 'ruRU', 'Ты принес вооружение из деревни Черного Копыта?'),
(11173, 'ruRU', 'Удалось добыть яд? Морланн в любую минуту может спохватиться, что его не хватает...'),
(11169, 'ruRU', 'Какие успехи в изучении тотема?'),
(11150, 'ruRU', 'Ну что, хорошо разгорелось?'),
(11185, 'ruRU', 'Что там у тебя, $c?'),
(11174, 'ruRU', 'Ну как, тебе уже удалось покончить со слизнюками?'),
(11158, 'ruRU', '<Брогг с надеждой смотрит на вас.>$B$BТвоя приносить Броггу перья?'),
(11160, 'ruRU', 'Принес знамя Каменного Молота?'),
(11161, 'ruRU', 'Принес сущности? Они нужны Броггу, чтобы отомстить за друзей!'),
(11217, 'ruRU', 'Ты нашел, где растет змейкин хвост?'),
(11159, 'ruRU', 'Брогг должен знать, принес ли ты покой свирепым духам Каменного Молота?'),
(11162, 'ruRU', 'Брогг свершит свою месть!'),
(11152, 'ruRU', 'Вы возложили венок?');

DELETE FROM `quest_offer_reward_locale` WHERE `ID` IN (1178,1111,1112,1114,1287,11143,11144,11148,11149,11173,11169,11156,11150,11185,11151,11174,11184,11160,11161,11217,11159,11162,11152,11212,11211) AND `locale` = 'ruRU';
INSERT INTO `quest_offer_reward_locale` (`ID`, `locale`, `RewardText`) VALUES
(1178, 'ruRU', 'Вас прислал Поззик? Когда-то ведь он был одним из самых уважаемых механиков в Нижней шахте, но потом уволился и занялся гонками.$B$BПервое, что он сделал, придя в команду, – разработал две новых модели двигателей. Дела Союза ремонтников могут пострадать без Поззика, но ему уже нет до этого дела... Гонки для него все!'),
(1111, 'ruRU', 'Ах, да! Детали для Крейвела. Хорошо, что за ними пришли! Этот хлам валяется тут уже давно, я даже начал бояться, что он проржавеет!'),
(1112, 'ruRU', 'Ура, детали! Надо отнести их гномам. Но сначала...$B$BВынем оттуда кое-что...'),
(1114, 'ruRU', 'Вот они! Мои гидромеханизмы и усилительные болты! То, что нужно. Большое спасибо!'),
(1287, 'ruRU', 'Я пошлю лейтенанта Келдвелла разобраться с дезертирами в Разрушенной башне. Помимо этого, я поручу ему отыскать Рита. Все-таки болото не такое уж большое и непроходимое, что он не сможет найти его.'),
(11143, 'ruRU', 'Капитан Ваймс считает, что племя Зловещего Тотема повинно в преступлении, совершенном в таверне "Последний привал"? Я бы не ставил их первыми в списке подозреваемых, но раз капитан Ваймс прислал тебя для дальнейшего расследования, должно быть, у него были на то причины. В любом случае, племя Зловещего Тотема давно у нас как заноза в пятке, и мы всегда рады принять любую помощь.'),
(11144, 'ruRU', '<Капитан Дарилл просматривает боевой план.>$B$BЯ бы сказал, что все яснее ясного. Они берут на себя уничтожение людских поселений в этих краях, и я не сомневаюсь, что следующим в списке будет Терамор.'),
(11148, 'ruRU', '<Капитан Дарилл осматривает принесенное вами из деревни Черного Копыта вооружение.>$B$BСомневаться не приходится – вооружение в точности соответствует описанию. Но дело в том, что оно ни разу не побывало в бою. Должно быть, деревня Черного Копыта – не единственный аванпост племени Зловещего Тотема в топях.'),
(11149, 'ruRU', 'Значит, ты хочешь, чтобы я предсказала будущее племени Зловещего Тотема? А если я скажу, что нужды в том нет?'),
(11173, 'ruRU', '<Ученик Гарион изучает яд.>$B$BПросто прекрасные образцы! Он и не отличит.'),
(11169, 'ruRU', '<Вы подробно рассказываете о ходе эксперимента.>$B$BТуман и молния? И все? Но ведь этот тотем должен вызывать еще какие-то эффекты! Тотем бы не стали нести на поле брани, не будь у него каких-то полезных свойств.'),
(11156, 'ruRU', '<Ученик Морланн кивает.>$B$BЭто хорошо. И не могу сказать, что буду страдать без их общества. А то даже в сердце трясин уединения не найдешь!'),
(11150, 'ruRU', 'Надеюсь, племя Зловещего Тотема усвоило урок. Попытаются вернуться – мы их встретим не только факелом. Мои ученики помогут мне дать отпор.'),
(11185, 'ruRU', '<Анделло читает письмо, потрясенный его содержанием.>$B$BМы давно подозревали, что Орда и Отрекшиеся в союзе, и это письмо подтверждает наши подозрения. При помощи магии и оружия Отрекшихся племя Зловещего Тотема станет еще более страшным врагом. Баланс сил в Калимдоре может сильно измениться.'),
(11151, 'ruRU', '<Капитан внимательно выслушивает ваш доклад.>$B$BМы отомстили за Джеймса и его семью, но это их не вернет.'),
(11174, 'ruRU', 'Ну, похоже, там и правда стало немного почище...$B$BПолагаю, пора заняться делом и основательно порыться в обломках.'),
(11184, 'ruRU', 'Хвала небесам, наконец кто-то откликнулся на это объявление! Я уж думал, что придется утроить награду за голову этого дурацкого ящера, а наш бюджет такого не потянет!'),
(11160, 'ruRU', 'Брогг тебе благодарен, $n. У Брогга большие планы на клановое знамя.'),
(11161, 'ruRU', 'Ты получаешь благодарность Брогга, $n. Как только Брогг отдает свою магию тотему, тотем освобождает духов убитых друзей Брогга.'),
(11217, 'ruRU', 'Спасибо тебе! Теперь наши парни продержатся в поле, пока Мокси не доставит груз, оставшийся после крушения дирижабля.'),
(11159, 'ruRU', 'Брогг благодарит тебя за помощь, хотя и знает, что от властителя Мок\'Морокка ты не дождешься слов благодарности. Но теперь, по крайней мере, выжившие члены клана могут наконец заняться насущными делами, и мертвые им больше досаждать не будут. А Броггу осталось сделать одну вещь.'),
(11162, 'ruRU', '<Брогг слушает ваш рассказ о том, что произошло у входа в Логово Ониксии.>$B$BБрогг так и знал, что Ониксия побоится биться с тобой. Все черные драконы нападают, только когда победа однозначна.$B$BБрогг благодарен тебе за помощь, $N. Он никогда не забудет, что ты для него сделал. Брогг никогда не отступит от своей цели вернуть Каменный Молот на родину!'),
(11152, 'ruRU', '<Капитан Ваймс мягко улыбается, слушая ваш рассказ о событиях, произошедших у памятника семейству Хьяль.>$B$BПриятно знать, что они все видят. Ты же не рассказал им о Джеймсе, правда?$B$BЯ отправлю ему послание жены, хотя сомневаюсь, что он поймет.'),
(11212, 'ruRU', 'Хорошо, что Терамор прислал кого-то. Подозрения вашего друга-охранника были верны. У нас есть новые соседи, и они не из тех, кто приносит запеканку.'),
(11211, 'ruRU', 'Хорошо, что Табета прислала тебя. Неплохо было бы, если б она передала каких-то припасов, пока сюда не доставят груз с разбившегося дирижабля, но, как мне известно, сейчас ей просто нечем поделиться. Если у тебя есть вкус к приключениям, то я назначил награду за одного особенно противного ящера.');

```

Branch:
3.3.5

TC rev. hash/commit: https://github.com/TrinityCore/TrinityCore/commit/c08f086a02729f3b7e4c99f4741063e45301ebf8

TrinityCore rev. 2021-06-25 18:43:42 +0300 (3.3.5 branch) (Win64, Release, Static)

**Comment by offl** [comment]
Are you using only wowhead to find texts?

**Comment by Freeman256** [comment]
Not only wowhead, but also https://wowroad.info, and when I can't find the Russian text (which is extremely rare) I use the google translator. Yes, my corrections are not 100% canon, but it is significantly better than nothing.

**Comment by Freeman256** [comment]
I meant that first of all I look for texts on wowhead, if they are not there - on wowroad, if they are not there too - google translator + a little common sense.

**Comment by offl** [comment]
I asked because I just noticed that you already updated the title of the quest ID=11173 earlier
I found ancient locale texts for quests from 2008 or earlier and texts of that quest matches https://wowroad.info/?quest=11173 but not tbc.wowhead
Then I checked quest 1178 and both wowhead and wowroad but the texts were the same, so I'm a bit confused now

Yep, I recommend to first use wowhead as primary source, then wowroad because I used it long time ago for some stuff and from what I know its content actually was not copied from wowhead

**Comment by Freeman256** [comment]
Yes, I may have already updated the title of this quest, sorry about that.
Let me share with you my thoughts on the discrepancy between the texts on WoWhead and WoWroad: it seems to me that in 2008 Blizzard made the text that is now on WoWhead and which was at that time on WoWhead, then the times of the Lich King passed, and "Blizzard marked these quests as obsolete "and they are no longer present in the game. Then, with the release of WOW Classic, they returned these quests, but since they actually pulled the old locations and quests onto the Legion client, they decided to translate the quests into Russian.
I don't really want to conduct an investigation to establish the absolute truth, so I use the indicated sources, and if I see that the text in Russian does not look correct (even if it is taken from WoWhead), I change it in such a way that it would look nice on Russian, and, of course, that he was as canonical as possible.

**Comment by offl** [comment]
> Then, with the release of WOW Classic, they returned these quests, but since they actually pulled the old locations and quests onto the Legion client, they decided to re-translate the game into Russian.

I heard something like that but haven't found lots of differences, moreover I compared our creature names with ancient locale texts and got 1800 mismatches. Ancient locales matches tbc.wowhead btw

All our current locale texts are not from wowhead, they were bruteforced from MoP
Sometimes they requires changes but there's no reason to try to fix all small mismatches simply because we are not entirely sure which version is more correct for wotlk. In other words better to change existing text only if it's really necessary(if english text is completely different, for example if quest had 3 objectives in WotLK but it was changed to 2 in Cata). Just in case if you notice that tons of things are translated differently

https://github.com/rudb/rudb/blob/master/mangos.sql
Those are texts you see in wowroad, as you can see they were added before wowroad was created

**Comment by Freeman256** [comment]
I completely agree: I will make the most canonical translation, and if suddenly someone in the future thinks that my translation is not accurate enough or completely incorrect, he can always use the update command.

Regarding rudb - in my second update of Russian localization (part 2), I just tried to extract from my database those values ​​that I found on the Internet and which I installed on top of vanilla trinity core, and then publish them here. And, unfortunately, it turned out that this database was not much more complete than on a clean server.
